### DISCUSSION TITLE
Maxar ARD sample data

### DISCUSSION THREAD

**Discussion by drwelby** [gjoseph92/stackstac#54] [Ideas]
Hi from Maxar!

We have some COG gridded imagery in a public bucket that could be fun to play with:

`s3://maxar-ard-samples/v3`

[Directory browser](https://ard.maxar.com/samples#v3/)

In short, these are COGs plus STAC on a quadkey UTM grid, and are quadkey-addressable to blocks and even to pixels. For more info there's docs [here](https://ard.maxar.com/docs/).

Our team's previous project used Dask and we've thought about building Dask arrays with these images, so I'm interested to see how this project works with our STAC data.

We don't yet have Raster Extension metadata up to speed, so you'll need to set up the stack something like:

```python
cat = Catalog.from_file('https://ard.maxar.com/samples/v3/rio_deforestation/order_collections/5763013822349948994_root_collection.json')
stack = stackstac.stack(cat, chunksize=512, epsg=32610, assets=['visual'], resolution=0.30517578125)
```

**Comment by drwelby** [comment]
With the above example we seem to be missing some bands, the "visual" imagery is RGB but loads to:

```
<xarray.DataArray 'stackstac-eaac4c5d9fdd6c3718ac94fee49663f7' (time: 19, band: 1, y: 17409, x: 17488)>
```

I haven't had time to investigate, this could just be due to our metadata structure, or I'm missing a keyword.

**Reply by kylebarron** [reply]
Seems like you're hitting this from the `README`:

> **Single-band raster data only!** Each band has to be a separate STAC asset—a separate `red`, `green`, and `blue` asset on each Item is great, but a single `RGB` asset containing a 3-band GeoTIFF is not supported yet.

**Reply by drwelby** [reply]
ah there it is, thanks!
